src/BNO055.cpp

```diff
@@ -0,0 +1,151 @@
+#include <BNO055.hpp>
+
+IMU::BNO055::BNO055(IO::I2C& i2C): i2c(i2C) { }
+
+bool IMU::BNO055::setup() {
```

Comment:
Both methods here need a lot of inline documentation explaining the weird I2C stuff that's going on and what each step is doing